test/cypress/components/filterable-list-control.js

```diff
@@ -0,0 +1,75 @@
+export class FilterableListControl {
+
+  filterableListControl() {
+    return cy.get( '.o-filterable-list-controls' );
+  }
+
+  filterableElement( selector ) {
+    return this.filterableListControl().get( selector );
+  }
+
+  filterableListElement( name ) {
+    return cy.get( `#o-filterable-list-controls_${ name }` );
+  }
+
+  openFilterableListControl() {
+    return cy.get( '#o-filterable-list-controls' )
+      .find( 'button' ).first().click( { force: true } );
+  }
+
+  filterItemName( name ) {
+    return this.filterableListElement( 'title' ).type( name );
+  }
+
+  filterFromDate( date ) {
+    return this.filterableListElement( 'from-date' ).type( date );
+  }
+
+  filterToDate( date ) {
+    return this.filterableListElement( 'to-date' ).type( date );
+  }
+
+  results() {
+    return cy.get( '.o-post-preview_content' );
+  }
+
+  firstResult() {
+    return this.results().first();
+  }
+
+  lastResult() {
+    return this.results().last();
+  }
+
+  openExpandable() {
+    return cy.get( '.o-filterable-list-controls .o-expandable' )
+      .click( { force: true } );
+  }
```

Comment:
Can we get rid of this and replace uses of it with `showFilter()`?

Reply:
Yes I removed `openExpandable` and replaced with `showFilter`